app/scripts/presenters/visit_presenter.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I eventually settled on using dashes on the server, but these kind of niceties can wait till the initial dust settles ^_^